Boop/Boop/scripts/JoinLinesWithSpace.js

```diff
@@ -0,0 +1,15 @@
+/**
+	{
+		"api":1,
+		"name":"Join Lines With Space",
+		"description":"Joins All Lines With a Space",
+		"author":"riesentoaster",
+		"icon":"table",
+		"tags":"join, space",
+        "bias": -0.1
```

Comment:
whitespace for bias and the rest of the data is inconsistent (part 2)

Reply:
Fixed the whitespace/tab issue.